### ISSUE TITLE
Confirmation Modal

### ISSUE THREAD

**Issue by sfmskywalker** [elsa-workflows/elsa-core#3080]

**Comment by jdevillard** [comment]
Do we plan to add a input text to validate the name of the workflow or yes (for bulk) delete operation? 

Or a flag to enable this kind of feature ?

**Comment by sfmskywalker** [comment]
Yes to the workflow name input validation, but I suggest we create a separate issue for that.
A flag to control this sounds nice. There are probably other aspects as well that we might want to let the developer control.
